sdk/sdk-view/src/test/java/mil/emp3/api/GeoJSONExportTest.java

```diff
@@ -213,16 +215,15 @@ private void parseGeoJSON(String featureType, String geoJSONStr, Feature feature
                         Logger.info(featureType + " is rendered as Polygon");
                         Logger.info(feature.toString());
                         Logger.info(obj.toString());
-                        compareFeatureToPolygon(feature, (Polygon) obj);
+                        compareFeatureToGeoJsonPolygon(feature, (Polygon) obj);
                     } else if (obj instanceof Point && featureType.equals("Text")) {
-                        String str1 = ((Text) feature).getText();
-                        String str2 = ((Text) obj).getText();
-                        Assert.assertEquals(str1, str2);
+                        // GeoJSON has no place for text, so comparison is omitted
                     } else {
                         Logger.error(featureType + " is rendered as " + obj.getClass().getSimpleName());
                     }
                 }
             } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Suggest logging instead of print stack trace.

Reply:
I had added it during debugging, its gone now.